packages/desktop-client/src/components/schedules/SchedulesTable.tsx

```diff
@@ -193,7 +193,6 @@ export function SchedulesTable({
       <Row
         height={ROW_HEIGHT}
         inset={15}
-        backgroundColor="transparent"
```

Comment:
`Row` doesn't have this property. This is detected after switching the file to tsx.